float64 start_x
float64 start_y
float64 start_orientation_angle
geometry_msgs/Point[] check_points
